fenix/app/src/main/java/org/mozilla/fenix/debugsettings/tabs/TabTools.kt

```diff
@@ -161,6 +195,97 @@ private fun TabCountRow(
+    onAddTabsToInactiveClick: ((Int) -> Unit),
+    onAddTabsToPrivateClick: ((Int) -> Unit),
+) {
+    var tabQuantityToCreate by rememberSaveable { mutableStateOf(DEFAULT_TABS_TO_ADD) }
+    var hasError by rememberSaveable { mutableStateOf(false) }
+    val keyboardController = LocalSoftwareKeyboardController.current
+
+    Column {
+        Text(
+            text = stringResource(R.string.debug_drawer_tab_tools_tab_creation_tool_title),
+            color = FirefoxTheme.colors.textPrimary,
+            style = FirefoxTheme.typography.headline5,
+        )
+
+        TextField(
+            value = tabQuantityToCreate,
+            onValueChange = {
+                tabQuantityToCreate = it
+                hasError = it.isEmpty() || !it.isDigitsOnly() || it.toInt() == 0
+            },
+            modifier = Modifier.fillMaxWidth(),
+            textStyle = FirefoxTheme.typography.subtitle1,
+            label = {
+                Text(
+                    text = stringResource(R.string.debug_drawer_tab_tools_tab_creation_tool_text_field_label),
+                    color = FirefoxTheme.colors.textPrimary,
+                    style = FirefoxTheme.typography.caption,
+                )
+            },
+            isError = hasError,
+            keyboardOptions = KeyboardOptions(
+                keyboardType = KeyboardType.Number,
+            ),
+            keyboardActions = KeyboardActions(
+                onDone = {
+                    keyboardController?.hide()
+                },
+            ),
+            singleLine = true,
+            colors = TextFieldDefaults.textFieldColors(
+                textColor = FirefoxTheme.colors.textPrimary,
+                backgroundColor = Color.Transparent,
+                cursorColor = FirefoxTheme.colors.borderFormDefault,
+                errorCursorColor = FirefoxTheme.colors.borderWarning,
+                focusedIndicatorColor = FirefoxTheme.colors.borderPrimary,
+                unfocusedIndicatorColor = FirefoxTheme.colors.borderPrimary,
+                errorIndicatorColor = FirefoxTheme.colors.borderWarning,
+            ),
+        )
+
+        Spacer(modifier = Modifier.height(8.dp))
+
+        PrimaryButton(
+            text = stringResource(id = R.string.debug_drawer_tab_tools_tab_creation_tool_button_text_active),
+            enabled = !hasError,
+            onClick = {
+                onAddTabsToActiveClick(tabQuantityToCreate.toInt())
+            },
+        )
+
+        Spacer(modifier = Modifier.height(8.dp))
+
+        if (inactiveTabsEnabled) {
+            PrimaryButton(
+                text = stringResource(id = R.string.debug_drawer_tab_tools_tab_creation_tool_button_text_inactive),
+                enabled = !hasError,
+                onClick = {
+                    onAddTabsToInactiveClick(tabQuantityToCreate.toInt())
+                },
+            )
+
+            Spacer(modifier = Modifier.height(8.dp))
+        }
```

Comment:
Wondering if this should maybe be last, since it's the only conditional element

Reply:
My intent here was to mirror the ordering of the tab counts. Inactive tabs are really just a subset of normal tabs, so I wanted to keep them visually next to each other, if that makes sense.